codacy/values-microk8s.yaml

```diff
@@ -53,6 +53,12 @@ engine:
       cpu: 1
       memory: 2Gi
 
+codacy-tools:
+  resources:
+    limits:
+      cpu: 0.5
```

Comment:
Maybe we can reduce this. Current usage is cpu:17m, mem:424MiB, so I would use cpu: 200m, and mem: 512MiB  or something like that. The mem and cpu requirement for microk8s are very tight.